user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -133,12 +133,30 @@ settings.
 ### Getting to the boot screen
 
 Just after you power on your machine, make the Qubes OS medium available to the
-computer by inserting your DVD or USB drive. Shortly after the Power-on
-self-test (POST) is completed, you should be greeted with the Qubes OS boot
-screen. 
+computer by inserting your USB drive.
+
+Then you'll need to power on the computer and immediately press a specific key to
+enter what's called the "BIOS" of you computer. The key to press varies wildly from
+one computer to another. `Esc`, `Del` and `F10` are
+common keys. If you're not sure about the key to press you'll have to view in the
+machine's user guide or serch the web for `accessing BIOS <COMPUTER_MODEL>` 
+(replacing with your specific computer model).
+
+The BIOS menu will looks something like the following pictures:
+
+| ![](/attachment/doc/Thinkpad-t430-bios-main.jpg)  | |
+|----|----|
+| [CC-BY-SA-4.0 Vitaly Zdanevich](https://commons.wikimedia.org/wiki/File:Thinkpad-t430-bios-main.jpg) | ADD here UEFI
```

Comment:
You can put this copyright information in [COPYRIGHT.md](https://github.com/QubesOS/qubes-attachment/blob/master/COPYRIGHT.md) instead.

Also, it's good practice to include something in the title rather than leaving it blank (i.e., between the square brackets in the `![]` part).

I'm also not sure why we're using a table here and what "ADD here UEFI" means. If that's supposed to be a placeholder for a future image, maybe make it an HTML comment or something.

Also, don't forget to submit a PR against https://github.com/QubesOS/qubes-attachment/tree/master with your image(s).

Reply:
Didn't know about COPYRIGHT.md. Will put it there.

"ADD here UEFI" --> because of github's non-existant drafting UI.

I'll also fix the table to make the title "Legacy BIOS" and "UEFI" and will add the image on qubes-attachment.